recipes/dftbplus-python/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "dftbplus" %}
+{% set version = "20.1" %}
+
+package:
+  name: {{ name|lower }}-python
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/69405b705c9d8e8130c0eaf2daff0906ff0a33df.tar.gz
+  sha256: 70e1ae26570d05b174f6092d7e8f409747a701451cbde081ad147a25895382a9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install tools/pythonapi -vv"
+  skip: True  # [win]
```

Comment:
Can this be `noarch: python` please?

Reply:
Probably yes, but how is than Windows case handled without a `dftbplus` version available?

Reply:
It just wouldn't be installable.  noarch python only needs to build on linux